### ISSUE TITLE
remove mint with balance from list

### ISSUE THREAD

**Issue by KKA11010** [cashubtc/eNuts#250]
**Is your feature request related to a problem? Please describe.**
"In my wallet I have a mint (my old one) which no longer exists. The app does not let me remove the mint because i have a tiny balance on it; now my (usable) balance is slightly off."

**Describe the solution you'd like**
Provide the dangerous possibility to remove a mint and the associated token

**Describe alternatives you've considered**
Alternative solution would be to move the mint and the token to a "unsued" table and exclude it from the balance calculation. It could also be recovered if needed.

**Comment by acurious** [comment]
I encountered the same problem. I transferred my usable balance to a different wallet on a separate device. After that, I performed a clean installation and then transferred the funds back. This resulted in losing all transaction history; however, the balance issue was successfully resolved.

**Comment by KKA11010** [comment]
> I encountered the same problem. I transferred my usable balance to a different wallet on a separate device. After that, I performed a clean installation and then transferred the funds back. This resulted in losing all transaction history; however, the balance issue was successfully resolved.

Yes I would say this is currently a good temporary workaround if someone wants to clean up their mint list and balance.
